frontend/src/pages/project/Overview.vue

```diff
@@ -8,7 +8,15 @@
                     <tr class="border-b">
                         <td class="w-1/4 font-medium">Editor</td>
                         <td>
-                            <a v-if="editorAvailable" :href="project.url" target="_blank" class="forge-button-secondary py-1 mb-1"><span class="ml-r">{{project.url}}</span><ExternalLinkIcon class="w-4 ml-3" /></a>
+                            <div v-if="editorAvailable">
+                                <div v-if="isVisitingAdmin" class="my-2">
```

Comment:
This suggests in your "after" screenshot, we should be seeing the `project.url`, but I do not?

Reply:
Unless I'm missing something, that's the `http://localhost:7881`